Tests/URKArchiveTestCase.m

```diff
@@ -352,11 +352,11 @@ - (NSURL *)archiveWithFiles:(NSArray *)fileURLs name:(NSString *)archiveName arg
 {
     NSURL *textFile = [self randomTextFileOfLength:100000];
     
-    // Generate multi-volume archive, with parts no larger than 20 KB in size
+    // Generate multi-volume RAR 4.0 archive, with parts no larger than 20 KB in size
     NSString *commandOutputString = nil;
     [self archiveWithFiles:@[textFile]
                       name:baseName
-                 arguments:@[@"-v20k", @"-vn"]
+                 arguments:@[@"-v20k", @"-ma4", @"-vn"]
```

Comment:
This change need by updating rar executable binary.

`-vn` option can not use in RAR 5.0 format.
`-ma4` means rar generates RAR 4.0 archive format.

> Creating old style volume names in archive.r00, archive.r01 format is not supported.
https://www.winrar-france.fr/winrar_instructions_for_use/source/html/HELPRAR5Format.htm